youtube_dl/extractor/esquire.py

```diff
@@ -0,0 +1,98 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .adobepass import AdobePassIE
+from ..utils import (
+    extract_attributes,
+    smuggle_url,
+    update_url_query,
+)
+
+
+class EsquireIE(AdobePassIE):
+    _VALID_URL = r'https?://tv\.esquire\.com/now/(?:[^/]+)/?(?:full-episode)?/(?P<title>[^/?#]+)/(?P<id>[^/?#]+)'
+    _TESTS = [{
+        'url': 'http://tv.esquire.com/now/team-ninja-warrior/full-episode/finals-week-1/631550531649',
+        'md5': '436ee8095d7179704cf2738994d36a20',
+        'info_dict': {
+            'id': '631550531649',
+            'ext': 'mp4',
+            'title': 'Finals Week 1',
+            'description': 'The first finals episode features the winners of 3 qualifying episodes.',
+            'upload_date': '20160301',
+            'timestamp': 1456808400,
+            'uploader': 'NBCU-MPAT',
+        }
+    }, {
+        'url': 'http://tv.esquire.com/now/friday-night-tykes/full-episode/if-you-wanna-show…/903098435679',
```

Comment:
if the test case has the same flow as the previous one then there is no need for another test.